src/utils/index.ts

```diff
@@ -5,6 +5,6 @@ export { getColorByCoordinates } from "./getColorByCoordinates";
 export { getCoordinatesByColor } from "./getCoordinatesByColor";
 export { getHueByCoordinates } from "./getHueByCoordinates";
 export { getCoordinatesByHue } from "./getCoordinatesByHue";
-export { isValidHEX } from "./isValidHEX";
+export { isValidHEX } from "./isValidHex";
```

Comment:
```suggestion
export { isValidHEX } from "./isValidHEX";
```
I think you should change the name of the file, not the name of its export.

Reply:
@Wondermarin fixed